src/main/windows/editor.js

```diff
@@ -69,6 +69,10 @@ class EditorWindow extends BaseWindow {
 
     let win = this.browserWindow = new BrowserWindow(winOptions)
     this.id = win.id
+    if (isOsx) {
+      win.setBackgroundColor('#00000000')
+      win.setVibrancy('ultra-dark')
+    }
```

Comment:
@Jocs @ZhuZhengyi  I don't know about this feature but these values should only be set when the theme is active, right?

Reply:
> I tested this PR, it works well under the macOS system, but as @fxha said, it has no effect on Windows and Linux systems, and is consistent with one-dark, so I suggest that vibrancy-dark theme As a custom theme, we will support the custom theme in the v0.17.0. We also want to recommend vibrancy-dark as the official custom theme on our official website, but we may not merge this PR now, @ZhuZhengyi what's your opinion?

@Jocs ok!

Reply:
@fxha yes, setVibrancy only macOS electron API.